package.json

```diff
@@ -38,7 +38,7 @@
     "babel-plugin-transform-decorators-legacy": "^1.3.4",
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
-    "babel-preset-stage-2": "^6.5.0",
+    "babel-preset-stage-2": "^6.18.0",
```

Comment:
Could make sense to bump all the presets together. AFAIK they are part of a monorepo and get bumped as a group. From luma.gl

    "babel-cli": "^6.18",
    "babel-core": "^6.18",
    "babel-preset-es2015": "^6.18",
    "babel-preset-stage-2": "^6.18",